Seminar9/task_rand_cvs/Deco.py

```diff
@@ -0,0 +1,47 @@
+# Напишите следующие функции:
+# Нахождение корней квадратного уравнения
+# Генерация csv файла с тремя случайными числами в каждой строке. 100-1000 строк.
+# Декоратор, запускающий функцию нахождения корней квадратного уравнения с каждой тройкой чисел из csv файла.
+# Декоратор, сохраняющий переданные параметры и результаты работы функции в json файл.
+
+import csv
+import json
+import os
+from typing import Callable
+
+
+# передача аргументов в функцию из файла
+def start_quadro_fromcsv(func: Callable):
+    def wrapper(*args):
+        with open('rand_int.csv', 'r', newline='', encoding='utf-8') as f_read:
+            res = []
+            csv_file = csv.reader(f_read)
+            for i, line in enumerate(csv_file):
+                if i != 0:
+                    a,b,c = map(int, line)
+                    res = func(a,b,c)
+            return res
```

Comment:
По условию возврат не обязателен. Просто вызов
```suggestion
                    res = func(*map(int, line))
            return res
```